fe/fe-core/src/main/java/com/starrocks/qe/ResultReceiver.java

```diff
@@ -125,7 +125,7 @@ public RowBatch getNext(Status status) throws TException {
             LOG.warn("fetch result execution exception, finstId={}", finstId, e);
             if (e.getMessage().contains("time out")) {
```

Comment:
Why `time out` instead of `timeout` ?

Reply:
> Why `time out` instead of `timeout` ?

this meesage is generated by brpc, like 'Ocurrs time out with specfied time xxx MILLISECONDS'